image-viewer.py

```diff
@@ -56,72 +59,87 @@ def render(self):
 
         self.root.bind("<KeyPress>", self.onKeyPress)
         self.root.mainloop()
-        
+
     def create_menu(self):
         menubar = Menu(self.root)
         self.root.config(menu=menubar)
-        
+
         file_menu = Menu(menubar, tearoff=False)
-        file_menu.add_command(label='Open Folder', command=self.open_folder, underline=0)
-        file_menu.add_command(label='Refresh', command=self.refresh, underline=0)
-        file_menu.add_command(label='Exit', command=self.root.destroy, underline=1)
-        menubar.add_cascade(label='File', menu=file_menu, underline=0)
-    
+        file_menu.add_command(label="Open File(s)", command=self.open_files)
+        file_menu.add_command(label="Open Folder", command=self.open_folder, underline=0)
+        file_menu.add_command(label="Refresh", command=self.refresh, underline=0)
+        file_menu.add_command(label="Exit", command=self.root.destroy, underline=1)
+        menubar.add_cascade(label="File", menu=file_menu, underline=0)
+
+    def open_files(self):
+        files = filedialog.askopenfilenames(
+            filetypes=(
+                ("all files accepted", ["*.png", "*.jpg", "*.ico", "*.gif", "*.jpeg"]),
+                ("png files", "*.png"),
+                ("jpg files", "*.jpg"),
+                ("jpeg files", "*.jpeg"),
+                ("gif files", "*.gif"),
+                ("ico files", "*.ico"),
+            )
+        )
+        self.image_files = []
+        self.image_files.append(*files)
```

Comment:
I get an error on this line (`self.image_files.append(*files)`) when I select multiple images: 
`TypeError: list.append() takes exactly one argument (2 given)` (2 being the number of images I selected).

I reckon this should be a quick and easy fix though, rather than passing `*files` into `append`, could do a for loop, something along the lines on:
```
for file in *files:
    self.image_files.append(file)
```

That would ensure that we're only passing one file in at a time.

Reply:
There's also a similar thing when Cancel is pressed:
`TypeError: list.append() takes exactly one argument (0 given)`
but I think the above for loop should fix that too.